TesseractOCR/G8Tesseract.mm

```diff
@@ -51,6 +51,22 @@ @implementation G8Tesseract
 
 @synthesize absoluteDataPath=_absoluteDataPath;
```

Comment:
do we? Actually, if there is a readwrite property, we need at least one setter to be called some where in our code. But there is no one, since you did ask for using ivars in the initializers. So in such case we'll have one uncoverred line in the code, which is readwrite property declaration.
There are some workarounds for this. One is @synthesise. Another is using setter in the initializer with the rewadwrite property.
Can you suggest something else?